test/xkeyboard-config-test.py.in

```diff
@@ -222,7 +259,8 @@ def run(combos, tool, njobs, keymap_output_dir):
 
     failed = False
     with multiprocessing.Pool(njobs) as p:
-        results = p.imap_unordered(tool, combos)
+        results = p.imap_unordered(tool.run, combos)
+        invocation: Invocation
```

Comment:
if you type annotate the `create_progress_bar` function you shouldn't need this one, though admittedly that's more complicated to do

Reply:
Turns out it was not easy to annotate.